.travis.yml

```diff
@@ -3,4 +3,6 @@ rvm:
   - 2.0
   - 2.1
   - 2.2
+before_install:
+  - gem update bundler
```

Comment:
Just a bit of googling for kittens ;)
On 22 Apr 2016 5:17 p.m., "Jeremy Kallman" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/maxwell/screencap/pull/35#discussion_r60804388>:
>
> > @@ -3,4 +3,6 @@ rvm:
> >    - 2.0
> >    - 2.1
> >    - 2.2
> > +before_install:
> > +  - gem update bundler
>
> Nice! How did you figure that out?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/maxwell/screencap/pull/35/files/0478476c539d753644599f4f679ba0ab546f2f99..d5b8a0e17ecd40a5fa38faaa71e08d916d591995#r60804388>
>